ci/env/install-minimal.sh

```diff
@@ -36,8 +36,7 @@ eval "${WORKSPACE_DIR}/ci/ci.sh build"
 
 # Install test requirements
 python -m pip install -U \
-  pytest==7.0.1 \
-  numpy
```

Comment:
I don't have full context here but we might still want this one given numpy is used in some testing (the comment says "Install test requirements")

Reply:
This is only for minimal tests and I just removed numpy as a dependency for that -- I think that's the right move since it will make sure `pip install ray` works and keeps working without numpy. I don't think we lost anything in terms of test coverage :)

Reply:
Ah got it. Agreed we should drop it for minimal tests 👍